packages/config/config/devices/0x0384/ha-zw-5sab.json

```diff
@@ -230,7 +213,7 @@
 		{
 			"#": "15",
 			"label": "Pir Triggers The Correspondence Between The Value of The Basic Set And The PIR State",
-			"description": "Pir triggers the correspondence between the value of the Basic Set and the PIR state, when value is 0x00 PIR triggers send the Basic Set command 0xff PIR alarm release send the Basic Set command 0x00, otherwise PIR triggers send the Basic Set command 0x00 PIR alarm release send the Basic Set command 0xff",
+			"description": "When value is 0x00 PIR triggers send the Basic Set command 0xff PIR alarm release send the Basic Set command 0x00, otherwise PIR triggers send the Basic Set command 0x00 PIR alarm release send the Basic Set command 0xff",
```

Comment:
This can be greatly shortened, preferably
label: "Motion: Basic Set Value"
no description
options:
- "0xff on triggered, 0x00 on cleared" (0)
- "0x00 on triggered, 0xff on cleared" (1)